editor.planx.uk/src/@planx/components/Feedback/Public/Public.tsx

```diff
@@ -21,14 +26,39 @@ import { Feedback, FormProps } from "../model";
 import { StyledToggleButtonGroup } from "../styled";
 
 const FeedbackComponent = (props: PublicProps<Feedback>): FCReturn => {
+  const handleSubmitFeedback = async (
+    values: FormProps,
+    { resetForm }: FormikHelpers<FormProps>,
+  ) => {
+    const metadata = await getInternalFeedbackMetadata();
+    const feedback = {
+      userComment: values.feedback,
+      feedbackScore: values.feedbackScore,
+    };
+    const data = {
+      ...metadata,
+      ...feedback,
+      feedbackType: "component" as FeedbackView,
+    };
+    const submitFeedbackResult = await insertFeedbackMutation(data).catch(
+      (err) => {
+        console.error(err);
+      },
+    );
+    props.handleSubmit?.(makeData(props, values, "_feedback"));
```

Comment:
I'd recommend making this `_feedback` key a bit more visible / discoverable

```ts
const PASSPORT_FEEDBACK_KEY = "_feedback" as const;
...
props.handleSubmit?.(makeData(props, values, PASSPORT_FEEDBACK_KEY));

```

As noted in the PR description, we'll want to re-use this value for the `ALLOW_LIST` for analytics.